docs/nlu/multi-skill-assistants.rst

```diff
@@ -0,0 +1,62 @@
+:desc: Iterate quickly by developing reusable building blocks of AI assistant skills
+       and combining them at training time.
+
+.. _multi_skill_bots:
+
+
+Multi Skill Assistants
+======================
+
+Rasa supports building AI assistants from multiple contextual AI assistant projects.
+This allows it to develop reusable building blocks of skills which you can reuse among
+your projects. One skill could for example handle chitchat, while the other skill
+is responsible for greeting your users. You can develop each skill isolated and then
+import them for the combined training of contextual AI assistant.
+
+An example directory structure could e.g. look like this:
+
+.. code-block:: bash
+
+    .
+    ├── config.yml
+    └── data
+        ├── GreetBot
+        │   ├── data
+        │   │   ├── nlu.md
+        │   │   └── stories.md
+        │   └── domain.yml
+        └── MoodBot
+            ├── config.yml
```

Comment:
Should MoodBot have its own config? If yes, shouldn't GreetBot have one too?

Reply:
I see that maybe the only thing in moodbot's config is importing greetbot -- if this is the case then we should make it clear that nothing else (e.g. policies, pipeline) goes in the `config.yml` for the lower projects

Reply:
I agree - will add this